pkgs/os-specific/linux/atop/default.nix

```diff
@@ -9,31 +19,52 @@ stdenv.mkDerivation rec {
     sha256 = "nsLKOlcWkvfvqglfmaUQZDK8txzCLNbElZfvBIEFj3I=";
   };
 
-  buildInputs = [zlib ncurses];
+  nativeBuildInputs = if withAtopgpu then [ python3.pkgs.wrapPython ] else [ ];
+  buildInputs = [ zlib ncurses ] ++ (if withAtopgpu then [ python3 ] else [ ]);
+  pythonPath = if withAtopgpu then [ python3.pkgs.pynvml ] else [ ];
 
   makeFlags = [
-    "SCRPATH=$out/etc/atop"
-    "LOGPATH=/var/log/atop"
-    "INIPATH=$out/etc/rc.d/init.d"
-    "SYSDPATH=$out/lib/systemd/system"
-    "CRNPATH=$out/etc/cron.d"
-    "DEFPATH=$out/etc/default"
-    "ROTPATH=$out/etc/logrotate.d"
+    "DESTDIR=$(out)"
+    "BINPATH=/bin"
+    "SBINPATH=/bin"
+    "MAN1PATH=/share/man/man1"
+    "MAN5PATH=/share/man/man5"
+    "MAN8PATH=/share/man/man8"
+    "SYSDPATH=/lib/systemd/system"
+    "PMPATHD=/lib/systemd/system-sleep"
   ];
 
+  patches = [
+    ./atop-pm.sh.patch
```

Comment:
I guess I agree with the auto-linting that adding a bunch of patches would be clearer with a comment in the place of use

Reply:
I have shuffled around the patches into fix-paths.patch for atop and netatop each (replacing FHS references with @@ templates), and commented the rest